client/src/components/App.jsx

```diff
@@ -118,22 +118,22 @@ class App extends React.Component {
 
     return (
       <div>
-        <div>
-          <Filter
-            onClickOption={this.handleOnChange}
-            currentCategory={state.currentCategory}
-            onClickMenu={this.handleOnClickMenu}
-            isVisible={state.isVisible}
-          />
-        </div>
-        <div id="container">
+        <Filter
+          onClickOption={this.handleOnChange}
+          currentCategory={state.currentCategory}
+          onClickMenu={this.handleOnClickMenu}
+          isVisible={state.isVisible}
+        />
+        <div id="container" style={{ display: 'flex' }}>
           <div className={left} onClick={this.handleOnLeftClick}>
             <svg height="50" width="50">
               <circle id="circle1" cx="25" cy="25" fill="#18181b" r="25" />
               <text fill="#9147ff" fontFamily="Arial" fontSize="30px" x="35%" y="68%">{rightArrow}</text>
             </svg>
           </div>
-          {state.videos.map((video) => <Video video={video} />)}
+          <div id="scrollMe" style={{ display: 'flex', overflow: 'hidden' }}>
```

Comment:
This is already on style.css.

Reply:
Not sure what you mean? One of the problems I ran into was the arrows responding to the scrolling. In other words, when I would scroll right, the arrow would move too.  So I had to isolate the arrows from the videos, so I made the new "scrollMe" tag.

Reply:
Sorry I selected the wrong line. `I meant line 127 <div id="container" style={{ display: 'flex' }}>`